lib/Memecoin.ts

```diff
@@ -1310,6 +1296,69 @@ export class MemecoinSDK {
     return BigInt(result.toFixed(0))
   }
 
+  private determinePoolOfMemecoin(memecoin: HydratedCoin): GetTokenPoolResponse {
+    if (!memecoin.dexInitiated) {
+      return { poolType: TokenPoolType.BondingCurve, poolFee: 0 }
+    }
+
+    switch (memecoin.dexKind) {
+      case 'univ2':
+        return { poolType: TokenPoolType.UniswapV2, poolFee: 0 }
+      case 'univ3':
+      case 'univ3-bonding':
+        return { poolType: TokenPoolType.UniswapV3, poolFee: 10000 }
```

Comment:
don't hardcode the poolFee. fetch it

Reply:
why do we fetch it? it is our memecoin right, we want them to go through our 10000 fee pool